src/components/Scorecards/ScorecardDetailsPage/ScorecardsTableCard/ScorecardsTableRow.tsx

```diff
@@ -1,97 +0,0 @@
-/*
- * Copyright 2022 Cortex Applications, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-import React from 'react';
-import { ScorecardServiceScore } from '../../../../api/types';
-import { makeStyles, TableCell, TableRow } from '@material-ui/core';
-import Box from '@material-ui/core/Box';
-import { Gauge } from '../../../Gauge';
-import { ScorecardServiceRefLink } from '../../../ScorecardServiceRefLink';
-import { ScorecardLadderLevelBadge } from '../../../Common/ScorecardLadderLevelBadge';
-import { humanizeAnyEntityRef } from '../../../../utils/types';
-import { defaultComponentRefContext } from '../../../../utils/ComponentUtils';
-
-const useStyles = makeStyles({
-  root: {
-    '& > *': {
-      borderBottom: 'unset!important',
-    },
-    height: '35px',
-  },
-  openIcon: {
-    width: '35px',
-  },
-  progress: {
-    height: '64px',
-    marginRight: '16px',
-  },
-});
-
-interface ScorecardsTableRowProps {
-  scorecardId: number;
-  score: ScorecardServiceScore;
-}
-
-export const ScorecardsTableRow = ({
-  scorecardId,
-  score,
-}: ScorecardsTableRowProps) => {
-  const classes = useStyles();
-  const currentLevel = score.ladderLevels?.[0]?.currentLevel;
-
-  return (
-    <React.Fragment>
-      <TableRow className={classes.root}>
-        <TableCell>
-          <ScorecardServiceRefLink
```

Comment:
One UX side effect of this change is that the entire row is no longer a clickable entity to go to a service details page, but instead the link is limited to the service name.

Reply:
this is definitely preferred!